### PR TITLE
Modified

### PR DESCRIPTION
**Pull request by Arshavin86**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+  "description": "",
+  "author": "Andrei",
+  "license": "",
+  "engines": {
+    "node": ">=6.13.0"
+  },
+  "scripts": {
+  },
+  "dependencies": {
+    "bluebird": "^3.5.3",
```
[the remark this answers is not in the document]
Comment (Arshavin86) [reply package.json]: Probably, not.

File: package.json [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+    "body-parser": "^1.17.0",
+    "express": "^4.15.0",
+    "react": "^15.4.2",
```
[the remark this answers is not in the document]
Comment (Arshavin86) [reply package.json]: > FYI react and react-dom are version 16 now
I renewed it to 15.6.2. Version 16 is not found.

[the remark this answers is not in the document]
Comment (therobinkim) [reply package.json]: react is version 16 https://reactjs.org/versions; would recommend looking into it further

[the remark this answers is not in the document]
Comment (Arshavin86) [reply package.json]: Fixed.

File: package.json [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+    "react-router-dom": "^4.3.1",
+    "cors": "^2.8.5",
+  },
+  "devDependencies": {
+    "eslint-config-hackreactor": "git://github.com/reactorcore/eslint-config-hackreactor",
+    "babel-core": "^6.23.1",
+    "babel-loader": "^6.3.2",
+    "babel-preset-es2015": "^6.22.0",
+    "babel-preset-react": "^6.23.0",
+    "webpack": "^2.2.1"
```
[the remark this answers is not in the document]
Comment (Arshavin86) [reply package.json]: Fixed.

File: package.json [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+    "react": "^15.4.2",
+    "react-dom": "^15.4.2",
+    "react-router-dom": "^4.3.1",
+    "cors": "^2.8.5",
+  },
+  "devDependencies": {
+    "eslint-config-hackreactor": "git://github.com/reactorcore/eslint-config-hackreactor",
+    "babel-core": "^6.23.1",
+    "babel-loader": "^6.3.2",
+    "babel-preset-es2015": "^6.22.0",
```
[the remark this answers is not in the document]
Comment (Arshavin86) [reply package.json]: Fixed.

**Pull request merged**